app/models/manageiq/providers/container_manager/container_template.rb

```diff
@@ -0,0 +1,67 @@
+class ManageIQ::Providers::ContainerManager::ContainerTemplate < ContainerTemplate
```

Comment:
I would expect them to be unrelated.  They may share code via a shared module, but other than that, they are independent.